openrouteservice/.gitignore

```diff
@@ -5,15 +5,15 @@ TODO.txt
 .classpath
 .project
 #/WebContent/WEB-INF/ors-config.json
-*.iws
-*.ipr
 cgiar-cache
 srtm_38_03.zip
-logs/
```

Comment:
why is this line being removed?
logs shouldn't be checked into github?

 src/test/resources/logs/TEST_LOGGING is explicitly un-ignored, so this line could stay?

Reply:
It's already ignored in the .gitignore of parent dir. If I leave it in here, than the negating rule  for TEST_LOGGING.js does not work.